commands/01-mod.js

```diff
@@ -146,10 +146,10 @@ const Module = new Augur.Module()
     const bannerHighRole = msg.member.roles.cache.filter(r => r.id != "281135201407467520").sort((a, b) => b.comparePositionTo(a)).first();
     const mentions = /<@!?(\d+)>/ig;
     const reason = suffix.replace(mentions, "").trim() || "[Member Ban]: Violating the Code of Conduct";
+
+    let members = new u.Collection();
     let match;
-    let banCount = 0;
-    let confirm = await u.confirm(msg, `Are you sure you want to ban the following?\n${msg.mentions.members.map(m => m.displayName).join("\n")}`);
-    while ((match = mentions.exec(suffix)) && confirm) {
+    while (match = mentions.exec(suffix)) {
       userId = match[1];
       try {
         let member = await msg.guild.members.fetch(userId);
```

Comment:
`.catch()` in case they're not present. Throws an error otherwise.
```suggestion
        let member = await msg.guild.members.fetch(userId).catch(u.noop);
```